clang/test/SemaSYCL/lb_sm_90.cpp

```diff
@@ -1,5 +1,4 @@
-// RUN: %clang_cc1 -internal-isystem %S/Inputs %s -triple nvptx64-nvidia-cuda -target-cpu sm_90 -fsycl-is-device -fsyntax-only -Wno-c++23-extensions -verify
-// expected-no-diagnostics
+// RUN: %clang_cc1 -internal-isystem %S/Inputs %s -triple nvptx64-nvidia-cuda -target-cpu sm_90 -fsycl-is-device -fsyntax-only -Wno-c++23-extensions -verify -S
 
 // Maximum work groups per multi-processor, mapped to maxclusterrank PTX
 // directive, is an SM_90 feature, make sure that no warnings/errors are issued.
```

Comment:
Could you please update comment in this test?

Reply:
Sure, done in: https://github.com/intel/llvm/pull/11851 Thank you!